examples/quick_cluster.py

```diff
@@ -0,0 +1,50 @@
+from functools import partial
+
+import trio
+import tractor
+
+
+async def sleepy_jane():
+    uid = tractor.current_actor().uid
+    print(f'Yo i am actor {uid}')
+    await trio.sleep_forever()
+
+
+async def main():
+    '''
+    Spawn a flat actor cluster, with one process per
+    detected core.
+
+    '''
+    portal_map: dict[str, tractor.Portal]
+    results: dict[str, str]
+
+    # look at this hip new syntax!
+    async with (
+
+        tractor.open_actor_cluster(
+            modules=[__name__]
+        ) as portal_map,
+
+        trio.open_nursery() as n,
+    ):
+
+        for (name, portal) in portal_map.items():
+            n.start_soon(
+                partial(
```

Comment:
Yah, I had the `partial()` originally because I thought we needed to pass a `name='blah'` kwarg but we're actually auto-generating them unless a sequence is passed.

I'm wondering if we should tweak this api to allow passing in *only* the `names: list[str]` and if so then the helper makes only as many actors as there are names? Also, in the case where `names` **is** passed we probably shouldn't be doing this `<parentname>.<childname>` generation right?